nestkernel/connection.h

```diff
@@ -257,13 +257,18 @@ Connection< targetidentifierT >::check_connection_( Node& dummy_target,
   // this line might throw an exception
   source.send_test_event( dummy_target, receptor_type, get_syn_id(), true );
 
-  // 2. does the target accept the event type sent by sourc
+  // 2. does the target accept the event type sent by source
   // try to send event from source to target
   // this returns the port of the incoming connection
   // p must be stored in the base class connection
   // this line might throw an exception
   target_.set_rport( source.send_test_event( target, receptor_type, get_syn_id(), false ) );
 
+  // 3. do the events sent by source mean the same thing as they are
+  // interpreted in target?
+  if ( !( source.sends_signal() & target.receives_signal() ) )
```

Comment:
I would add a comment here explaining that this really is a bitwise and, not a misspelled boolean and. Bitwise operations are so rare in NEST that code-readers might get confused easily.